src/App.tsx

```diff
@@ -1,22 +1,30 @@
 import { FileEntry } from '@tauri-apps/api/fs';
 import * as React from 'react';
 import { useCallback } from 'react';
+import { pdfjs } from 'react-pdf';
 import { Panel, PanelGroup, PanelResizeHandle } from 'react-resizable-panels';
 import { useDebounce } from 'usehooks-ts';
 
 import { EditorAPI } from './types/EditorAPI';
+import { PdfViewerAPI } from './types/PdfViewerAPI';
 import { ReferenceItem } from './types/ReferenceItem';
 import { AIView } from './views/AIView';
 import { CenterPaneView } from './views/CenterPaneView';
 import { FoldersView } from './views/FoldersView';
 import { ReferencesView } from './views/ReferencesView';
 
+pdfjs.GlobalWorkerOptions.workerSrc = new URL(
+  'pdfjs-dist/build/pdf.worker.min.js',
+  import.meta.url,
+).toString();
+
```

Comment:
I think that this should be moved outside the `App.tsx` file. Can we move it to the `PdfViewer`?

Reply:
Yes, sorry, it was there because of my initial experiments with the library and I forgot to move it afterwards. Thanks for the catch!